docs/repair.md

```diff
@@ -17,7 +17,14 @@ To repair an ontology (with the name `need-of-repair.owl`) run the following com
 
 This will generate a new file `results/repaired.owl`. You can compare this with the original file (either using unix `diff` or [robot diff](diff)). If the changes that were made look good then you can simply replace the source file with the repaired file (`mv results/repaired.owl need-of-repair.owl`).
 
-By default, annotation axioms are not migrated to replacement classes. However, this can be enabled for a list of annotation properties passed either as arguments to `--annotation-property` or in a term file `--annotation-properties-file`:
+To use ROBOT solely for merging axiom annotations, you can use:
+
+    robot repair --input uberon_axiom_annotation_merging.owl \
+      --merge-axiom-annotations true \
+      --output results/uberon_axiom_annotation_merged.owl
+
+By default, annotation axioms are not migrated to replacement classes. 
```

Comment:
@balhoff what does this mean? I dont get this part of the repair method at all. This sentence also makes no sense to me.

Reply:
@balhoff ?

Reply:
If you use that option, you can automatically move xrefs from an obsolete class to its replacement.

Reply:
I have updated the documentation to make the command a bit clearer, but this particular part I still dont get it.

Lets say we have

```
A sub R some B
```

And B is deprecated.

When I "repair" my ontology I would assume this happens:

```
A sub R some X
```

where X is a replacement of B.

How does this relate to "fixing" anything about B? This seems like a totally different command to me (`robot deprecate` or similar).

Reply:
I guess it's "repairing" the ontology itself.